deepchem/data/datasets.py

```diff
@@ -954,6 +939,15 @@ def merge(datasets: Sequence[Dataset]) -> "NumpyDataset":
     return NumpyDataset(X, y, w, ids, n_tasks=y.shape[1])
 
 
+class Shard(object):
```

Comment:
Could you put a `_` at the start of the class name, and likewise for all the other classes you pulled up to the top level?  These are meant to be internal classes, not for users to access directly.

Reply:
Fixed it!

Reply:
Thanks!  Can you do the same for the other classes too (TorchNumpyDataset, etc.)?  Those are also internal classes, not intended for direct use by users.